Wikimate.php

```diff
@@ -495,23 +567,87 @@ public function getError()
 /**
  * Models a wiki article page that can have its text altered and retrieved.
  *
- * @author  Robert McLeod
+ * @author  Robert McLeod & Frans P. de Vries
```

Comment:
Maybe update the copyright notice in the license file as well?

https://github.com/hamstar/Wikimate/blob/bc3214a56f8c21d48280a747e8aff69dad0810d8/LICENSE.md#L3

Reply:
Done.